pom.xml

```diff
@@ -129,6 +129,28 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>sonar-cloud</id>
+            <properties>
+                <!-- SonarQube rule exclusions for: Integration tests -->
+                <sonar.issue.ignore.multicriteria>e1,e2,e3,e4,e5</sonar.issue.ignore.multicriteria>
+                <!-- Disable: Public types, methods and fields (API) should be documented with Javadoc -->
```

Comment:
Reported as code smell with high severity, however we don't need to document test setups.